sse/src/main/scala/io/finch/sse/ServerSentEvent.scala

```diff
@@ -0,0 +1,47 @@
+package io.finch.sse
+
+import java.nio.charset.Charset
+
+import cats.Show
+import com.twitter.concurrent.AsyncStream
+import com.twitter.io.Buf
+import io.finch.{Encode, Ok, Output, Text}
+import io.finch.internal.{BufText, ToResponse}
+
+case class ServerSentEvent[A](
+  data: A,
+  id: Option[String] = None,
+  event: Option[String] = None,
+  retry: Option[Long] = None
+)
+
+object ServerSentEvent {
+  type EventStream[A] = AsyncStream[ServerSentEvent[A]]
```

Comment:
I'd vote for not introducing this type. `AsyncStean[ServerSentEvent[A]]` isn't that verbose and quite self-documenting.